recipes-kernel/linux/files/freedom-u540/0008-spi-nor-5.2-patch

```diff
@@ -0,0 +1,1587 @@
+diff --git a/Documentation/devicetree/bindings/mtd/cadence-quadspi.txt b/Documentation/devicetree/bindings/mtd/cadence-quadspi.txt
+index 4345c3a6f530..945be7d5b236 100644
```

Comment:
This needs to be a properly formatted patch.

Ideally this should also be a patch that has been sent upstream or is about to be sent upstream.

Reply:
It sounds like patches have been sent to the list, so that patch should be used instead